### DISCUSSION TITLE
Subfolders for Avatars/Emojis and actual Media

### DISCUSSION THREAD

**Discussion by No0negit** [Tyrrrz/DiscordChatExporter#796] [General]
Can you please add an ability (or even make it only 1 possible option) to download media in different subfolders?
All "technical" stuff related to the server, users (avatars), emojis to posts, etc in one folder.
Actual media (pics/vids) in another subfolder. Or maybe pictures in 1 subfolder and videos in 2 subfolder.

**Comment by 96-LB** [comment]
See #795 and #714 for discussion about this.
